### PR TITLE
Update Snapmaker Profiles

### PR DESCRIPTION
**Pull request by macdylan**
I own Snapmaker A350&J1, and here I have corrected and validated all the parameters for Snapmaker printers.

### CODE REVIEW

**Comment by hliebscher** [comment]
Currently @fr3ak2402 is helping consolidate the printer profile names.
Can you refer to this: https://github.com/SoftFever/OrcaSlicer/pull/1100#issuecomment-1556946352 to match the new name schema?

**Comment by macdylan** [comment]
image

@hliebscher Is this acceptable?

I noticed that filament no longer distinguishes between Nozzles in the new rules, and personally i think this unreasonable. Or did I miss something? 

in the Assets filenames, some are in lowercase while others are in uppercase, and this is a bit confusing to me.
image

**Comment by fr3ak2402** [comment]
@macdylan 

I have also considered different filaments for the nozzles, in the 2.x version of the profiles... At the time when I had set the new naming system, there were only 0.4 nozzles.

Since some things have changed for me, I have for example started to create all profiles with more nozzle sizes. I am also in the process of creating all the filaments for each nozzle. I will use the scheme from Bambulab with the `@base, @0.2 nozzle, @0.4 nozzle` and so on.

----------------------------------
Can you rename the J1 filaments, e.g. Snapmaker J1 PLA @0.4 nozzle etc.?

----------------------------------
In version 2.x the assets are also stored in a location in the profile, but you can leave that for now....

If you want, you can also create a PR in the OrcaSlicer Profile Library, you can find it under my repositories...

**Comment by SoftFever** [comment]
@macdylan
Thank you for the PR. This is fantastic.

There will be some issue when we rename a file or profile, it will cause issues for existing user profiles that were created based on the older names. Those profiles will no longer be able to be loaded if we change the names.

To address this, I will write up a guide about profiles for OrcaSlicer. Once that is complete, I will review the PR and provide my comments.

File: resources/profiles/Snapmaker/filament/Snapmaker Generic ABS.json
Comment (SoftFever) [inline resources/profiles/Snapmaker/filament/Snapmaker Generic ABS.json:1]: Deleting or renaming an existing preset will make all existing user presets based on it unusable.
Can we avoid renaming or deletion?

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Pull request closed without merging**